lib/game.rb

```diff
@@ -0,0 +1,48 @@
+require_relative '../lib/game_state'
+require_relative '../lib/game_presenter'
+require_relative '../lib/messages'
+require_relative '../lib/game_setting'
+require_relative '../lib/game_rules'
+
+class Game
+    include Validator
+    include Messages
+    include GameRules
+
+    attr_accessor :game_state, :game_presenter, :status
+
+    def initialize(game_presenter: ConsoleIO.new, board_size:, player_2:)
+        @game_presenter = game_presenter
+        @game_state = GameState.new(
+            player_2: player_2,
+            board_size: board_size
+        )
+        @status = :play
+    end
+
+    def play
+        4.times { turn() }
```

Comment:
why 4 times?

Reply:
this function is still a WIP -> i was testing TestIO amongst a few other things.